Administrator Guide/Split brain and ways to deal with it.md

```diff
@@ -0,0 +1,61 @@
+# Split brain and the ways to deal with it
+
+### Split brain:

```

Comment:
Typos: pristine, inconsistency, despite.... Please run the entire contents throu'  a spell checker once.